source/frontend/src/components/maps/leaflet/Map.tsx

```diff
@@ -290,13 +290,6 @@ const Map: React.FC<React.PropsWithChildren<MapProps>> = ({
                     ))}
                   </LayerGroup>
                 )}
-                {!!layerPopup && (
-                  <LayerPopup
-                    ref={popupRef}
-                    layerPopup={layerPopup}
-                    onViewPropertyInfo={onViewPropertyClick}
-                  />
-                )}
```

Comment:
I'm not sure this is the right fix. This would remove the layer popup for **every** map interaction. I think we still want to show popups on map click, right @devinleighsmith ?

Reply:
correct, probably what we need to do is prevent event propagation when clicking on the layer tree.

Reply:
@buddy326 does the above give you enough to try and attempt this fix again?